sphinx/types.rst

```diff
@@ -1,104 +1,170 @@
 .. types:
 
-Project Data
-==================
-
-New data imported into the openoa toolkit can take advantage of the data structures in the `types` module.
-The base class is openoa.types.PlantData, which represents all known pieces of data about a given wind plant.
-
-
-Schemas
+Plant Data
+##########
+
+This is the core data class used to contain all data relevant to a wind plant and is used throughout OpenOA.
+The PlantData holds multiple Pandas Data Frames, each with a specified schema.
+You can take advantage of the data structures in the :py:mod:`plant` module by instantiating it using one of the available constructors.
+
+:py:class:`openoa.plant.PlantData` is an attrs
+dataclass, and each of the core representations are Pandas :py:class:`DataFrame`s, e.g., :py:attr:`openoa.plant.PlantData.scada`,
+:py:attr:`openoa.plant.PlantData.reanalysis`, etc.
+
+The benefit of adopting attrs for the core structure is many data checks can happen automatically
+at initialization, so many of the :py:meth:`openoa.plant.PlantData.prepare` steps a user might implement, will already be addressed.
```

Comment:
The concept of an openoa.plant.PlantData.prepare() method is confusing. Maybe just say "so many data preparation steps a user might implement…"

Reply:
I've reworded this section to just describe the v3 interface without any references to v2 so it should be much more clear now.